libs/sdk-ui-dashboard/src/presentation/widget/insight/configuration/InsightAlertConfig/utils.ts

```diff
@@ -490,6 +595,62 @@ export function transformAlertByMetric(
     };
 }
 
+export function transformAlertByAttribute(
```

Comment:
Would be nice to document, what is purpose of this function / what exactly it does.

Reply:
Added some description to function